celery_utils/persist_on_failure.py

```diff
@@ -29,7 +29,7 @@ def on_failure(self, exc, task_id, args, kwargs, einfo):
                 task_id=task_id,  # Fixed length UUID: No need to truncate
                 args=args,
                 kwargs=kwargs,
-                exc=_truncate_to_field(FailedTask, 'exc', repr(exc)),
+                exc=_truncate_to_field(FailedTask, 'exc', repr(exc).replace(',', '')),
```

Comment:
The exceptions is being converted into string here. In python 3.5, exception has trailing comma, which was removed in python 3.7 release. So for support for python 3.5 and 3.8, removing the comma (which will exist in the case of 3.5)